Basic Rules.txt

```diff
@@ -842,14 +842,16 @@ either by partially obscuring your silhouette, or by providing slight protection
 from projectiles. Full cover will do both to a better degree. "Partial Cover" typically
 covers 30-50% of your body, whereas "Full Cover" will cover closer to 60-80%. Crouching
 or going prone will also take better advantage of your surroundings and make you a 
-smaller target. Debuffs to enemies Miss Chance are as follows:
+smaller target. Debuffs to opponent's Miss Chances are as follows:

```

Comment:
Should be "opponents' Miss Chances" as there most often will be more than one. Difference is location of the apostrophe.

Reply:
good catch!